cfgov/v1/admin.py

```diff
@@ -38,12 +38,63 @@ def feedback_page_title(feedback):
 
 @admin.register(Feedback)
 class FeedbackAdmin(admin.ModelAdmin):
+
+    actions = admin.ModelAdmin.actions + ['export_selection_as_csv']
```

Comment:
I don't think you need to explicity add this to `admin.ModelAdmin.actions`; you should be able to just do `actions = ['export_selection_as_csv']`. The [Django code](https://github.com/django/django/blob/1.8.15/django/contrib/admin/options.py#L875) automatically pulls in actions from all levels of inheritance.